target/linux/layerscape/image/gen_layerscape_sdcard_img_head.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (C) 2016 OpenWrt.org
```

Comment:
This doesn't look correct.

Reply:
I have changed the copyright to "Copyright (C) 2017 NXP semiconductors".